pyprof/prof/linear.py

```diff
@@ -29,10 +29,15 @@ class Linear(OperatorLayerBase):
 	bias in bprop is a reduction and hence is 1 read.
 	'''
 
-    gemmKernels = ["gemm", "gemv", "dot_kernel", "splitKreduce_kernel", "reduce_1Block_kernel"]
+    gemmKernels = [
+        "gemm", "gemv", "dot_kernel", "splitKreduce_kernel",
+        "reduce_1Block_kernel", "cutlass"
```

Comment:
What prompted this addition? Did a user get an assertion?

Reply:
I got an assertion while running NCF with the new docker, which presumably has a newer cuda tool kit / cublas.